src/components/dictionaryConcepts/components/CreateConceptForm.jsx

```diff
@@ -100,7 +109,7 @@ const CreateConceptForm = (props) => {
               )
               }
 
-              { (props.concept.toString().trim() === 'Set')
+              { conceptIsASet(props.concept.toString().trim())
```

Comment:
In reference to the name conceptIsASet, have you tried to look around for the convention of boolean names? If yes, can you give examples that you have seen?

Reply:
Consulting my colleagues on this.

Reply:
Noted. Changed this name to `isSetConcept`.